src/Illuminate/Mail/Transport/SesTransport.php

```diff
@@ -10,6 +10,12 @@
 
 class SesTransport extends AbstractTransport
 {
+    /**
+     * Amazon SES has a limit on the total number of recipients per message.
+     * See Also: https://docs.aws.amazon.com/ses/latest/dg/quotas.html.
+     */
+    const recipientLimit = 50;
```

Comment:
This constant must be RECIPIENT_LIMIT because of coding convention.

Reply:
Fixed, thanks!